pkg/kubelet/volumemanager/reconciler/reconciler.go

```diff
@@ -106,143 +123,114 @@ func (rc *reconciler) Run(stopCh <-chan struct{}) {
 
 func (rc *reconciler) reconciliationLoopFunc() func() {
 	return func() {
-		// Unmounts are triggered before mounts so that a volume that was
-		// referenced by a pod that was deleted and is now referenced by another
-		// pod is unmounted from the first pod before being mounted to the new
-		// pod.
+		rc.reconciliationFunc()
 
-		// Ensure volumes that should be unmounted are unmounted.
-		for _, mountedVolume := range rc.actualStateOfWorld.GetMountedVolumes() {
-			if !rc.desiredStateOfWorld.PodExistsInVolume(mountedVolume.PodName, mountedVolume.VolumeName) {
-				// Volume is mounted, unmount it
-				glog.V(12).Infof("Attempting to start UnmountVolume for volume %q (spec.Name: %q) from pod %q (UID: %q).",
+		if time.Since(rc.timeOfLastReconstruct) > rc.reconstructDuration {
+			rc.reconstructFunc()
+		}
+	}
+}
+
+func (rc *reconciler) reconciliationFunc() {
```

Comment:
A function should only contain "Func" in it's name if it returns a func, for example `func (rc *reconciler) reconciliationLoopFunc() func()`. In this case, I agree, the `Func` suffix is unnecessary and confusing.